Library/Formula/libwebm.rb

```diff
@@ -0,0 +1,43 @@
+class Libwebm < Formula
+  desc "WebM container"
+  homepage "http://www.webmproject.org/code/"
+  url "https://github.com/webmproject/libwebm/archive/libwebm-1.0.0.26.tar.gz"
+  sha256 "1e4034e03836ac974bb078b11383a10306d63879b350a34fdd575ab44b132222"
+
+  head "https://chromium.googlesource.com/webm/libwebm", :using => :git
```

Comment:
Removed head, it uses Cmake instead of a plain Makefile and they changed the binaries files and some other stuff that makes this formula a bit more complex without giving back much.